### PR TITLE
Mk3 Romanian translation

### PR DESCRIPTION
**Pull request by Hauzman**
Ro translation update

### CODE REVIEW

File: lang/po/Firmware_ro.po [hunk 1]
```diff
@@ -809,7 +809,7 @@ msgid ""
 "G-code sliced for a different level. Please re-slice the model again. Print "
 "cancelled."
 msgstr ""
-"Versiune de G-code incorecta. Va rugam pregatiti modelul din nou. Print "
+"Versiune de G-code incorecta. Va rugam pregatiti modelul din nou. Print"
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Missing space after `.Print`
```
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |G-code sliced for a |
   2 |different level.    |
   3 |Please re-slice the |
   4 |model again. Print  |
   5 |cancelled.          |
 translated text:
      ₀₁₂₃₄₅₆₇₈₉₀₁₂₃₄₅₆₇₈₉
   1 |Versiune de G-code  |
   2 |incorecta. Va rugam |
   3 |pregatiti modelul   |
   4 |din nou.            |
   5 |Printanulat.        |
   ```

File: lang/po/Firmware_ro.po [hunk 2]
```diff
@@ -1507,8 +1507,7 @@ msgstr "Curatati varful pentru calibrare. Apasati butonul cand terminati."
 msgid ""
 "Please insert filament into the extruder, then press the knob to load it."
 msgstr ""
-"Va rugam introduceti filamentul in extruder apoi apasati butonul pentru a-l "
-"incarca."
+"Va rugam introduceti filamentul in extruder apoi apasati butonul pentru a-l incarca."
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Message length exceeds 80 chars, please split to
```
"Va rugam introduceti filamentul in extruder apoi apasati butonul pentru a-l "
"incarca."
```

File: lang/po/Firmware_ro.po [hunk 3]
```diff
@@ -2151,8 +2150,7 @@ msgstr ""
 #: ../../Firmware/mmu2/errors_list.h:202 ../../Firmware/mmu2/errors_list.h:242
 msgid ""
 "The Selector cannot home properly. Check for anything blocking its movement."
-msgstr "Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza"
-"miscarea."
+msgstr "Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza miscarea."
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: please change to
```
msgstr ""
"Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza"
" miscarea."
```

File: lang/po/Firmware_ro.po [hunk 4]
```diff
@@ -2373,16 +2371,14 @@ msgstr ""
 #: ../../Firmware/ultralcd.cpp:3340
 msgid ""
 "XYZ calibration compromised. Left front calibration point not reachable."
-msgstr "Calibrarea XYZ compromisa. Punctele de calibrare din fata stanga nu pot fi"
-"atinse."
+msgstr "Calibrarea XYZ compromisa. Punctele de calibrare din fata stanga nu pot fi atinse."
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Same as above ... keep msgstr "" and don't exceed 80 chars per line please

File: lang/po/Firmware_ro.po [hunk 5]
```diff
@@ -2399,7 +2395,7 @@ msgstr ""
 #. MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_LEFT_FAR c=20 r=8
 #: ../../Firmware/ultralcd.cpp:3328
 msgid "XYZ calibration failed. Left front calibration point not reachable."
-msgstr ""
+msgstr "Calibrarea XYZ a esuat. Punctul de calibrare stanga fata nu poate fi atinsa."
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Same above

**Review by 3d-gussner — CHANGES_REQUESTED** [review]

File: lang/po/Firmware_ro.po (hunk 1 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: done

File: lang/po/Firmware_ro.po (hunk 2 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: done

File: lang/po/Firmware_ro.po (hunk 4 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: done

File: lang/po/Firmware_ro.po (hunk 3 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: done

File: lang/po/Firmware_ro.po [hunk 6]
```diff
@@ -584,16 +584,16 @@ msgid ""
...
 
 #. MSG_DESC_FINDA_DIDNT_TRIGGER c=20 r=8
 #: ../../Firmware/mmu2/errors_list.h:196 ../../Firmware/mmu2/errors_list.h:236
 msgid ""
 "FINDA didn't trigger while loading the filament. Ensure the filament can "
 "move and FINDA works."
 msgstr "FINDA nu sa declansat in timpul incarcarii"
-"Asigurativa ca filam. se misca si FINDA functioneaza"
-"
+"Asigurativa ca filamentul se misca si FINDA functioneaza"
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Again 
msgstr ""
and then the translation

File: lang/po/Firmware_ro.po [hunk 7]
```diff
@@ -2151,7 +2151,7 @@ msgstr ""
 #: ../../Firmware/mmu2/errors_list.h:202 ../../Firmware/mmu2/errors_list.h:242
 msgid ""
 "The Selector cannot home properly. Check for anything blocking its movement."
-msgstr "Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza"
+msgstr "Selector-ul nu poate reveni corect. Verificati daca exista ceva care blocheza "
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: Again
msgstr ""
then translation

Missing point at the end

File: lang/po/Firmware_ro.po [hunk 8]
```diff
@@ -2399,7 +2398,8 @@ msgstr ""
 #. MSG_BED_SKEW_OFFSET_DETECTION_FAILED_FRONT_LEFT_FAR c=20 r=8
 #: ../../Firmware/ultralcd.cpp:3328
 msgid "XYZ calibration failed. Left front calibration point not reachable."
-msgstr ""
+msgstr "Calibrarea XYZ a esuat. Punctul de calibrare stanga fata nu poate fi "
```
Comment (3d-gussner) [inline lang/po/Firmware_ro.po]: again
msgstr ""
missing point at the end

**Review by 3d-gussner — CHANGES_REQUESTED** [review]
Sorry but found few more.

Please check again for long translations if exceeding ~78 chars to split them. 
msgstr ""

File: lang/po/Firmware_ro.po (hunk 6 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: @3d-gussner 
like this below I see now I have for a few dont know how I change it 
msgstr ""
"FINDA nu sa declansat in timpul incarcarii"
"Asigurativa ca filamentul se misca si FINDA functioneaza"

File: lang/po/Firmware_ro.po (hunk 8 above)
Comment (Hauzman) [reply lang/po/Firmware_ro.po]: ther is in the row below 
"atinsa."

**Review by 3d-gussner — APPROVED** [review]

**Review by 3d-gussner — CHANGES_REQUESTED** [review]
Build failed as one message exceeds the hard limit.
https://app.travis-ci.com/github/prusa3d/Prusa-Firmware/jobs/586011916#L296-L308

**Review by 3d-gussner — APPROVED** [review]
Checked with `./lang-check.py` :heavy_check_mark:

**Pull request merged**